.github/workflows/build-and-push.yml

```
@@ -40,3 +49,4 @@ jobs:
           docker_context: ${{ matrix.dockerfile_path }}
           dockerfile_path: ${{ matrix.dockerfile_path }}
           tag: ${{ matrix.tag }}
+          suffix: ${{ matrix.suffix }}
```

```suggestion
          suffix: ${{ matrix.tag }}
```
It seems, that tag and suffix are the same.

---

No please not. When we will support Fedora build-and-push actions, suffixes will not be used at all.